apollo-router/src/plugins/telemetry/formatters/mod.rs

```diff
@@ -75,7 +108,7 @@ pub(crate) fn filter_metric_events(event: &tracing::Event<'_>) -> bool {
     })
 }
 
-pub(crate) fn to_map(resource: Resource) -> BTreeMap<String, serde_json::Value> {
+pub(crate) fn to_map(resource: Resource) -> HashMap<String, serde_json::Value> {
```

Comment:
do we need the attributes to stay in the same order everywhere?

Reply:
No I don't think it should be a requirement